README.md

```diff
@@ -105,3 +107,4 @@ See the “examples” folder for a more complete example.
 
 See examples
 
+Note: on windows ALT-0178 = ² degree symbol.
```

Comment:
This note is incorrect. The character "²" (U+00B2, `&#178;`) is not a degree symbol but a superscript two. The degree sign is "°" (U+00B0, `&#176;`).

It is also weird to have this note here. Why explain how to type this character on Windows, and not on MacOS and Linux? And why this character only? There are a few other characters in the README that are not in the ASCII table and therefore not directly accessible on many keyboard layouts. Namely:

* "−" (U+2212, `&#8722;`): minus sign
* "±" (U+00B1, `&#177;`): plus-minus sign
* "×" (U+00D7, `&#215;`): multiplication sign
* "α" (U+03B1, `&#945;`): Greek small letter alpha
* "π" (U+03C0, `&#960;`): Greek small letter pi
* "“" (U+201C, `&#8220;`): left double quotation mark
* "”" (U+201D, `&#8221;`): right double quotation mark

To me this looks like a personal note rather than something that truly documents the library.

Reply:
Yes is at least partly a personal note. You are right it should not be in there.
Will remove it.